### PR TITLE
Summary page - output

### PR DESCRIPTION
**Pull request by hungnguyendinh1999**
Just the output part of summary page.

Got the FE-BE calls to work. Other than that:
- UI changes to output-textbox. It's currently outputting the different "context" prompt.
- Prompting (Context) for different Vocabulary level
- Upload file working
- Added Read-out-loud button (not yet implemented)
- Minor changes otherwise

### CODE REVIEW

File: frontend/src/assets/read-speak.png
Comment (reesque) [inline frontend/src/assets/read-speak.png:1]: read-speak

New icon just dropped

File: server/index.js [hunk 1]
```diff
@@ -28,13 +28,18 @@ app.get('/', (req, res) => {
...
 
     if (!message || !context) {
         return res.status(400).send("No message or context provided");
     }
 
-    const response = await getGPTSummarizeResonse(message);
-    res.send(response.choices[0].message.content);
+    // const response = await getGPTSummarizeResponse(message);
+    // res.send(response.choices[0].message.content); // Send back to FE
+    res.send(context) // FOr testing purposes only
```
Comment (reesque) [inline server/index.js:42]: Is this experimental code...?

File: frontend/src/screen/SummaryScreen.tsx [hunk 2]
```diff
@@ -1,50 +1,151 @@
...
+    { level: 'Intermediate', instruction: 'Use moderately complex language for intermediate readers.' },
+    { level: 'Advanced', instruction: 'Use advanced language with technical details where appropriate.' },
+  ];
+
+const harmContext =
+    "If the input text contains harmful, illegal, or offensive content, respond with 'Content not allowed.' and give a 1-sentence explanation.";
+
+const constructContext = (selectedVocabLevelString: string) => {
+    const defaultContext =
+        "Summarize in a neutral and concise manner. " + harmContext;
```
Comment (reesque) [inline frontend/src/screen/SummaryScreen.tsx:29]: Check `openaiService.js` in server. The summarize prompt is already passed in, context here only need the harm context.

File: frontend/src/screen/SummaryScreen.tsx [hunk 3]
```diff
@@ -1,50 +1,151 @@
...
+    // get selectedVocabLevel from selectedVocabLevelString
+    const vocabLevel = vocabLevelArray.find(
+        (option) => option.level === selectedVocabLevelString
+    );
+    if (vocabLevel) {
+        return `${defaultContext} ${vocabLevel.instruction}`;
```
Comment (reesque) [inline frontend/src/screen/SummaryScreen.tsx:35]: Here, if you want, you can add another parameter in `getGPTSummarizeResonse` in server, and pass in another body param in client, so you don't have to concat too much stuff

**Review by reesque — CHANGES_REQUESTED** [review]

File: server/index.js (hunk 1 above)
Comment (hungnguyendinh1999) [reply server/index.js:42]: Yes. I don't want it to call Chat API every time any of you click.

Comment (reesque) [reply server/index.js:42]: Make another endpoint for testing then. It's a good idea to do that anyways.

**Review by reesque — APPROVED** [review]

**Pull request merged**